admin/upsell.php

```diff
@@ -1,17 +1,17 @@
 <?php namespace HM\BackUpWordPress; ?>
 <div class="hmbkp-upsell">
 
-<p>
-
 <?php
-/** translators:  the 1st placeholder is the first part of the anchor tag with the link to the extensions admin page and the second is the closing anchor tag */
-printf(
-	__( 'Store your backups securely in the Cloud with %1$sour extensions%2$s', 'backupwordpress' ),
-    '<a href="<?php echo esc_url( https://bwp.hmn.md/ ); ?>" target="_blank">',
-    '</a>'
+/** translators:  the 1st placeholder is the first part of the anchor tag with the link to the plugin review page and the second is the closing anchor tag */
+$cta_message = printf(
+	__( 'If you\'re finding BackUpWordPress useful, please %1$s rate it on the plugin directory%2$s.', 'backupwordpress' ),
+	'<a target="_blank" href="http://wordpress.org/support/view/plugin-reviews/backupwordpress">',
+	'</a>'
 );
 ?>
-
-</p>
-
+    <div id="hmbkp-cta-message" class="updated notice is-dismissible">
```

Comment:
Good point. I don't think I adjusted any of the tab/spacing stuff but I likely did without realizing. :(